packages/web-frontend/src/sagas.js

```diff
@@ -1006,29 +1001,62 @@ function getTimeZone() {
 }
 
 /**
- * Fetches drilldown data for a given view and level.
+ * Fetches enlarged dialog data for a given view, drillDown level and date range.
  */
-function* fetchDrillDownData(action) {
-  const { parameterLink, parameterValue, drillDownLevel, ...restOfAction } = action;
-  const drillDownData = yield call(
-    fetchViewData,
-    {
-      isExpanded: true,
-      extraUrlParameters: { [parameterLink]: parameterValue },
-      drillDownLevel,
-      ...restOfAction,
-    },
-    fetchDrillDownError,
-  );
-  if (drillDownData) {
-    yield put(
-      fetchDrillDownSuccess(drillDownLevel, { ...drillDownData, parameterLink, parameterValue }),
-    );
+function* fetchEnlargedDialogData(action) {
```

Comment:
Do you agree with consolidating the three sagas?

Reply:
Yes I think it is definitely an improvement